tests/30rooms/60version_upgrade.pl

```diff
@@ -386,18 +386,21 @@ sub upgrade_room_synced {
 
       my ( $pl_content, $new_room_id );
 
-      # Note that this test assumes that moderators by default are allowed to upgrade rooms
-
       matrix_join_room_synced(
          $upgrader, $room_id
       )->then( sub {
          # Make the joined user a moderator
          matrix_change_room_power_levels(
             $creator, $room_id, sub {
-               ( $pl_content ) = @_;
-               $pl_content->{users}->{$upgrader->user_id} = JSON::number(50);
-               log_if_fail "PL content in old room", $pl_content;
-            }
+            ( $pl_content ) = @_;
+            $pl_content->{users}->{$upgrader->user_id} = JSON::number(50);
+
+            # Note that this test assumes that moderators by default are allowed to upgrade rooms
+            # Change the PL rules to allow moderators to send tombstones
+            $pl_content->{events}->{"m.room.tombstone"} = JSON::number(50);
```

Comment:
I'd be tempted to instead have the creator upgrade tbh

Reply:
I should stick this in a comment, but the bug that this tested for was a moderator getting upgraded to an admin in the old room. So changing the upgrader to be an admin initially may not preserve the test case?

Reply:
hmm, Ok

Reply:
I'll add a comment.